x-pack/plugins/security_solution/public/management/cypress/tasks/isolate.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ActionDetails } from '../../../../common/endpoint/types';
+
+export const interceptActionRequests = (
+  cb: (responseBody: ActionDetails) => void,
+  alias: string
+) => {
+  cy.intercept('POST', '/api/endpoint/action/*', (req) => {
+    req.continue((res) => {
+      const {
+        body: { action, data },
+      } = res;
+
+      cb({ action, ...data });
+    });
+  }).as(alias);
+};
+
+export const sendActionResponse = (action: ActionDetails) => {
+  cy.task('sendHostActionResponse', {
+    action,
+    state: { state: 'success' },
+  });
+};
+
+export const isolateHostWithComment = (comment: string, hostname: string) => {
```

Comment:
Just wondering if all of these functions (from here down) that are "screen" specific should actually be placed into `screens/**` folder since (at least the way I understood it), `screens/` is where we would have the methods available to interact with them. I see some perhaps going into a `isolation_flyout` and other into a `alerts` file. 

_(FYI: In my pending PR, I have added a `screens/alerts.ts`, so if you create that, and depending on which one of us merges first, you might have some merge conflicts)_

Reply:
IMO its better to keep them per test. We usually end up with numerous separated helpers/selectors/actions that for the next person working with them are:
1. Really hard to find since these files tend to have hundreds of lines and dozens of different functions
2. Tricky in their reusability. Something that works in alerts context might not be reusable when adding alert from different context.